test/unit/master/test_subjob.py

```diff
@@ -36,9 +36,9 @@ def setUp(self):
             ],
         )
 
-    def test_set_atoms_subjob_id(self):
+    def test_subjob_constructor_sets_subjob_id_on_atoms(self):
         atoms = [Mock(), Mock()]
-        self._subjob._set_atoms_subjob_id(atoms, 4)
+        Subjob(1, 4, Mock(), Mock(), atoms)
```

Comment:
whoa, didn't even know you could use keyword arguments unless the function declaration looked like
def f(a=foo, b=bar)